hasjob/views/index.py

```diff
@@ -353,7 +349,7 @@ def index(basequery=None, md5sum=None, tag=None, domain=None, location=None, tit
 
     search_query = request.args.get('q')
```

Comment:
`search_query` should be a parameter to `index`, falling back to sourcing from the request.